torch/ao/quantization/fx/prepare.py

```diff
@@ -1137,7 +1108,8 @@ def insert_observers_for_model(
             node.name, (None, None, None, None, None))
         input_quantized_idxs: List[int] = prepare_custom_config.input_quantized_indexes
         output_quantized_idxs: List[int] = prepare_custom_config.output_quantized_indexes
-        node_name_to_target_dtype_info[node.name] = _get_target_activation_dtype_for_node(
+        # Dict[str, Optional[Tuple[Union[torch.dtype, type], bool]]]
```

Comment:
can you just put this as mypy type instead of a comment?

```
node.meta["target_dtype_info"] : Dict[str, Optional[Tuple[Union[torch.dtype, type], bool]]] = ...
```

Reply:
yeah I started with this, but it doesn't work

Reply:
maybe try splitting into two lines?

```
node_target_dtype_info : Dict[str, Optional[Tuple[Union[torch.dtype, type], bool]]] = ...
node.meta["target_dtype_info"] = node_target_dtype_info
```

Reply:
Or I think its fine with no comment too, just think if the return type changes the comment will be confusing

Reply:
OK, will try

Reply:
I think this is working, thanks